### PR TITLE
Adding documentation for control plane / data plane split, and completing airflow helpers migration

### PR DESCRIPTION
**Pull request by ezvz**
## Summary

Adding docs on control plane, and filling a gap in the airflow oss migration.

## Why / Goal

Improving documentation.

## Reviewers

@nikhilsimha @hzding621

### CODE REVIEW

**Review by nikhilsimha — APPROVED** [review]

File: airflow/oss_helpers.py [hunk 1]
```diff
@@ -0,0 +1,608 @@
...
+NO_SKIP_MODES = ["stats-summary", "label-join"]
+# For spark 3 migration into streaming client mode, we'll migrate per team.
+STREAMING_CLIENT_TEAMS = [
+    "zipline_test",
+    #    "claim_v21",
+    "china_trust_defense",
+    #    "cs_ds",
+    #    "psx",
+    #    "payments_risk",
+]
```
Comment (hzding621) [inline airflow/oss_helpers.py:27]: can we remove these

File: airflow/oss_helpers.py [hunk 2]
```diff
@@ -0,0 +1,608 @@
...
+    default_dag_owner_str = "one_airbnb_team_id_" + str(base.get("one_airbnb_team_id"))
+
+    if dag_owner_override:
+        override = team_conf["default"].copy()
+        override.update(team_conf.get(dag_owner_override, {}))
+        if override.get("one_airbnb_team_id"):
+            default_dag_owner_str = "one_airbnb_team_id_" + str(
+                override.get("one_airbnb_team_id")
+            )
+        user = override.get("user")
```
Comment (hzding621) [inline airflow/oss_helpers.py:48]: can we remove these?

File: airflow/oss_helpers.py [hunk 3]
```diff
@@ -0,0 +1,608 @@
...
+    """
+    Define the custom downstream tasks. Ex:
+        GroupBy:
+            upload -> Mussel export and upload.
+        Join Backfill:
+            upload join stats -> Mussel export and upload
+        Join Log Flattening:
+            upload log stats -> Mussel export and upload
+        Join Consistency Check:
+            upload consistency_check -> mussel export and upload
```
Comment (hzding621) [inline airflow/oss_helpers.py:317]: can we update the references to mussel?

File: airflow/oss_helpers.py [hunk 4]
```diff
@@ -0,0 +1,608 @@
...
+            input_table=f'{output_table(conf["metaData"])}_consistency_upload',
+            key_prefix="CHRONON_CONSISTENCY_METRICS_STATS_BATCH",
+            table_suffix="_ooc_stats",
+        )
+        encoded_export >> mussel_helper.upload_mussel_stats(
+            dag, conf, team_conf, table_suffix="_ooc_stats"
+        )
+        return encoded_export
+    if conf_type == "joins" and mode == "log-summary":
+        encoded_export = mussel_helper.create_mussel_stats_upload_export(
```
Comment (hzding621) [inline airflow/oss_helpers.py:354]: can we remove any references to mussel?

**Review by hzding621 — APPROVED** [review]

**Review by pengyu-hou — APPROVED** [review]

**Comment by pengyu-hou** [comment]
merge it as is

**Pull request merged**